xradar/io/backends/nexrad_common.py

```diff
@@ -0,0 +1,248 @@
+"""
+Data and functions common to all types of NEXRAD files.
+
+"""
+
+# The functions in this module are intended to be used in other
+# nexrad related modules. The functions are not and should not be
+# exported into the pyart.io namespace.
```

Comment:
Maybe rephrasing the provenience here, so occasional users see that this is xradar with functionality forked/vendored from Py-ART.